src/components/App/App.js

```diff
@@ -10,22 +10,35 @@ class App extends React.Component {
     super();
     this.state = {
       movies: [],
-      selectedDetails: null
+      selectedDetails: null,
+      error: false,
     };
   }
 
   componentDidMount() {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
       .then(response => response.json())
       .then(data => this.setState({movies: data.movies}))
-      .catch()
+      .catch(error => this.setState({error: true}))
   }
 
   selectMovie = id => {
-    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
-      .then(response => response.json())
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/:${id}`)
+      .then(response => {response.ok && response.json()})
```

Comment:
The error handling took us a bit to figure out but good work on problem-solving this together!